spec/lib/operation_id_spec.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: false
```

Comment:
ah, this is how you got it to pass.  is there any desire to address the frozen string literal issue upstream in grape, so that this can be set to "true"?